tests/test_flight_boundary.py

```diff
@@ -0,0 +1,23 @@
+from matplotlib.path import Path
+from SDA import*
+import unittest
+import numpy as np
+
+class FlightBoundaryTestCase(unittest.TestCase):
+
+    def setUp(self):
+
```

Comment:
Please remove this new line at the start of the method. Please do this for all methods